library/include/borealis/sidebar.hpp

```diff
@@ -45,8 +45,9 @@ class SidebarItem : public View
     std::string label;
     bool active = false;
 
-    Sidebar* sidebar     = nullptr;
-    View* associatedView = nullptr;
+    Sidebar* sidebar              = nullptr;
```

Comment:
I wonder if we can go even further. Remove SidebarItem entirely, add a single void* userdata in ListItem.

Use ListItem instead of SidebarItem in Sidebars, in the userdata put the three things we need: sidebar, associatedView, associatedViewUserData

Backport all features of ListItem in SidebarItem (if there is any?)

What do you think?